app/Console/Commands/ReportDevices.php

```diff
@@ -21,8 +22,13 @@ public function __construct()
         parent::__construct();
         $this->addArgument('device spec', InputArgument::OPTIONAL);
         $this->addOption('fields', 'f', InputOption::VALUE_REQUIRED, default: 'hostname,ip');
-        $this->addOption('output', 'o', InputOption::VALUE_REQUIRED, __('commands.report:devices.options.output', ['types' => '[table, csv, none]']), 'table');
+        $this->addOption('output', 'o', InputOption::VALUE_REQUIRED, __('commands.report:devices.options.output', ['types' => '[table, csv, json, none]']), 'table');
         $this->addOption('list-fields');
+        $this->addOption('list-relationships', '-L', InputOption::VALUE_NONE);
```

Comment:
Instead of using list-relationships, you could add support for bash completion.  (you should probably do that either way)

Reply:
This is a handy item for seeing what is available.

This is akin to suggestion `--list-fields` be removed.

Reply:
Honestly completion I think would be better left to another PR. Testing it and it seems broken in general currently for options like this. Does not work with `--fields` either.